server/.env

```diff
@@ -1,4 +1,4 @@
-DATABASE_URL="postgresql://shuru:password@postgres:5432/mql"
+DATABASE_URL="postgresql://shuru:password@localhost:5432/mql"
 TEST_DATABASE_URL="postgresql://shuru:password@localhost:5432/mql_test"
```

Comment:
This should be `TEST_DATABASE_URL="postgresql://shuru:password@localhost:5432/mql_test"`